processor/filterprocessor/metrics_test.go

```diff
@@ -546,7 +546,7 @@ func TestFilterMetricProcessorTelemetry(t *testing.T) {
 				IsMonotonic: true,
 				DataPoints: []metricdata.DataPoint[int64]{
 					{
-						Value:      4,
+						Value:      5,
```

Comment:
Out of curiosity: where is this from? Is it intentional?

Reply:
I believe this is because of https://github.com/open-telemetry/opentelemetry-collector/pull/11360, right?

Reply:
It is intentional, see https://github.com/open-telemetry/opentelemetry-collector/pull/11360#issuecomment-2393817108